layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -7,7 +7,7 @@
         "api_version": "@VK_VERSION@",
         "implementation_version": "@GFXRECONSTRUCT_VERSION@",
         "description": "GFXReconstruct Capture Layer Version @GFXRECONSTRUCT_VERSION_STRING@",
-        "introduction": "A software for capturing and replaying Vulkan API calls on Desktop systems",
+        "introduction": "The GFXReconstruct Capture Layer intercepts Vulkan API calls and logs them to a capture file. Other tools in the GFXReconstruct suite can inspect, modify, or replay capture files generated by this component.",
         "url": "https://vulkan.lunarg.com/doc/sdk/latest/windows/capture_tools.html",
```

Comment:
Here on top you can set globally `"platforms": [ "WINDOWS", "LINUX" ]` gfxrecon is not on macOS right?

Currently the settings and preset don't inherit the `platform` automagically but this is a feature I'll add.

Reply:
Actually it should probably be: `"platforms": [ "WINDOWS", "LINUX", "ANDROID" ],` instead